ci_support/upload_or_check_non_existence.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env python
```

Comment:
Where is this file coming from?

Reply:
conda-smithy

Reply:
No, it's not.

Reply:
Yeah, something weird is happening here. The __pycache__ directory with its `.pyc` shouldn't be getting added either.

Reply:
I think my local machine ended up in some weird stated based on the last time I was doing smithy development.